### ISSUE TITLE
Allow leading paragraphs in multi-paragraph items

### ISSUE THREAD

**Issue by wking** [spdx/license-list-XML#576]
Spun off from [here][1], now that #414 has been closed.  I'd really like to replace our [current][2]

```xml
      <item>
        <bullet>7.</bullet>
        If, as a consequence of a court judgment or allegation of patent
        …
        License would be to refrain entirely from distribution of the Program.
        <p>
          If any portion of this section is held invalid or
```

with:

```xml
      <item>
       <bullet>7.</bullet>
       <p>
          If, as a consequence of a court judgment or allegation of patent
          …
          License would be to refrain entirely from distribution of the Program.
        </p>
        <p>
          If any portion of this section is held invalid or
```

(or `<p><bullet>7.<bullet> If, …`) if we can get the styling worked out for that.  Semantically, the “If, as a consequence” bit is its own paragraph, and it would be nice to be able to mark that up.
[1]: https://github.com/spdx/license-list-XML/issues/414#issuecomment-320094253
[2]: https://github.com/spdx/license-list-XML/blob/cdb5b301a99c35d2aa83daa47c6ac4eba3a810df/src/GPL-2.0-only.xml#L283-L299

**Comment by bradleeedmondson** [comment]
I agree; we should have the first paragraph in each item enclosed in &lt;p&gt; tags.

I would actually go a step further and say that we should explicitly decide whether the schema allows for license text outside a paragraph tag or not. I believe we have been marking up XML with the assumption that mostly it is not allowed, but we haven't been enforcing this.

**Comment by wking** [comment]
On Fri, Dec 29, 2017 at 11:21:55AM -0800, Bradlee H. Edmndson wrote:
> I would actually go a step further and say that we should explicitly
> decide whether the schema allows for license text outside a
> paragraph tag or not.

I think it's going to be a bit more complicated than that (e.g. we
want to allow bare text inside crossRef), but I agree that we want to
move in this direction.  Having the XSD enforce this for us would be
great, and it would save us from having to land fixups like ce85ac90c
under pressure (although we'd still have a one-time cost to translate
the current content).

**Comment by goneall** [comment]
> although we'd still have a one-time cost to translate the current content

How much work would it be to translate the current content?

**Comment by wking** [comment]
On Fri, Dec 29, 2017 at 11:49:14AM -0800, goneall wrote:
> How much work would it be to translate the current content?

Probably a fair bit.  But we can chip away at it slowly; no need for a
flag day.

**Comment by swinslow** [comment]
Discussed on 2019-07-30 tech team call, re: whether to make paragraph tags mandatory within items. Decided not to make it mandatory. But cleaning these up would be a good first issue for new participants.

**Comment by swinslow** [comment]
This was noted in 2019 as a "good first issue" for cleanup for new participants. I don't see any follow-up of instances where this has been picked up. Is this something that is still of value for folks to put time towards? If not, I'd suggest closing this issue.
